source/functions/context.txt

```diff
@@ -154,7 +154,9 @@ values through the ``context.environment`` interface.
 Connect to a MongoDB Data Source or Third-Party Service (``context.services``)
 ------------------------------------------------------------------------------
 
-You can access a client for a linked MongoDB Atlas cluster or Data Lake through the ``context.services`` interface. You can also access third-party services, although this feature is deprecated. 
+You can access a client for a linked MongoDB Atlas cluster or
+{+adf-datasource+} through the ``context.services`` interface. You can
+also access third-party services, although this feature is deprecated.
```

Comment:
I wonder if, by this point, we should just leave off this last sentence? How long has it been deprecated now?

Reply:
I think that we should keep it until we fully deprecate services given that this is a reference page. Right now they're soft-deprecated, where existing services still work.